etc/insights-client.spec

```diff
@@ -136,8 +135,11 @@ if ! [ -d "/var/lib/insights" ]; then
 mkdir -m 644 /var/lib/insights
 fi
 
-# symlink the motd file
-ln -snf /etc/insights-client/insights-client.motd /etc/motd.d/insights-client
+#Link motd into place. Only do so if motd file doesn't exist
+if ! [ -L /etc/motd.d/insights-client ]; then
```

Comment:
This change depends on the companion change in __init__.py ... a previous version had it as an independent fix, but that's no longer the case here.

Reply:
And the reason is that we want the motd message to appear on upgrades ... just not reappear erroneously.